README.md

```diff
@@ -10,13 +10,13 @@
 
 ## Features
 
-*   **Write Once, Run Anywhere:** Airtest provides cross-platform APIs, including app installation, simulated input, assertion and so forth. Airtest uses image recognition technology to locate UI elements, so that you can automate games and apps without injecting any code. 
+*   **Write Once, Run Anywhere:** Airtest provides cross-platform APIs, including app installation, simulated input, assertion and so forth. Airtest uses image recognition technology to locate UI elements so that you can automate games and apps without injecting any code. 
```

Comment:
Removing the comma makes the sentence too long. Is it necessary to remove it?

Reply:
Yes it is necessary; because its improper to have a comma in that place.